e2e/wdio/platform/tests/date-picker.e2e-spec.ts

```diff
@@ -0,0 +1,243 @@
+import {
+    click,
+    elementArray, getAttributeByName, getCSSPropertyByName, getText, getValue,
+    refreshPage, scrollIntoView, sendKeys, setValue,
+    waitForElDisplayed, waitForPresent
+} from '../../driver/wdio';
+import { DatePicker } from '../pages/date-picker.po';
+import datePickerTestData from '../fixtures/testData/date-picker';
+
+describe('Date picker suite', function() {
+    const datePickerPage: DatePicker = new DatePicker();
+
+    beforeAll(() => {
+        datePickerPage.open();
+        waitForPresent(datePickerPage.inputDatePicker);
+    });
+
+    afterEach(() => {
+        refreshPage();
+        waitForPresent(datePickerPage.inputDatePicker);
+    });
+
```

Comment:
please add the retires like below:

beforeAll(() => {
        datePickerPage.open();
        waitForPresent(datePickerPage.inputDatePicker);
    }, 1);

    afterEach(() => {
        refreshPage();
        waitForPresent(datePickerPage.inputDatePicker);
    }, 1);